diff --git a/sentry/sentry.conf.example.py b/sentry/sentry.conf.example.py
index 826f43a..944dc63 100644
--- a/sentry/sentry.conf.example.py
+++ b/sentry/sentry.conf.example.py
@@ -48,7 +48,7 @@ DATABASES = {
         "NAME": "postgres",
         "USER": "postgres",
-        "PASSWORD": "",
+        "PASSWORD": "sentry",
-        "HOST": "postgres",
+        "HOST": "pgbouncer",
         "PORT": "",
     }
 }
